driverkit/misc/check-driver-job.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
```

Comment:
I could not find any link to this file. I guess it is intended as an example for the users, isn't it?

If yes, could we document it?

Reply:
This  yaml is used when you specify `CHECK_RUNTIME=kubernetes`, this creates a Job with the Docker image you noticed. This is transparent for users.